logzio/sender.py

```diff
@@ -47,33 +57,37 @@ def append(self, logs_message):
     def flush(self):
         self._flush_queue()
 
-    def _debug(self, message):
-        if self.debug:
-            print(str(message))
-
     def _drain_queue(self):
         last_try = False
 
         while not last_try:
-            # If main is exited, we should run one last time and try to remove all logs
+            # If main is exited, we should run one last time and try to remove
+            # all logs
             if not self.is_main_thread_active():
-                self._debug("Identified quit of main thread, sending logs one last time")
+                self.logger.debug(
+                    'Identified quit of main thread, sending logs one '
+                    'last time')
                 last_try = True
 
             try:
                 self._flush_queue()
 
+            # TODO: Which exception?
```

Comment:
Having some personal issues with leaving "TODO" in code.. please remove those, and lets open tickets for it